system/greenhouse-ccloud/ci/test-values.yaml

```diff
@@ -16,9 +16,10 @@ teams:
 teamRoleBindings:
   - teamRef: team1
     teamRoleRef: role1
-    clusterNames:
-      - cluster1
-      - cluster2
+    clusterSelector:
+      matchLabels:
+        cluster-type: compute
+        environment: production
```

Comment:
It might be more convenient to not distinguish right now until there's proper staging, QA, production. WDYT?